pipcompilemulti/environment.py

```diff
@@ -82,6 +88,9 @@ def outfile(self):
         return os.path.join(OPTIONS['base_dir'],
                             '{0}.{1}'.format(self.name, OPTIONS['out_ext']))
 
+    def is_package_in_outfile(self, pkg):
```

Comment:
Please use the context manager for `open` (since pip-compile-multi supports pypy too).
Also, I think it would be more user-friendly to compare lower-cased package names (e.g. Django vs django).
For the sake of robustness, it should handle the case when `outfile` doesn't exist (I realize such a command wouldn't make much sense, but I don't want to make it show cryptic traceback).
Another concern is that this method will re-read `outfile` for each `-P` option. It would be more efficient to read the file once, put all the lower-cased package names into a set and check each `-P` option in that set.
And last, but not least, please add an integration test, that will invoke this option similar to `tests/test_conflicts.py`.

Reply:
I have made the changes you requested. In addition I changed the --upgrade option's default to False since only one of --upgrade or --upgrade-package can be specified. This is also consistent with pip-compile. I believe all the automated tests are now passing with the exception of the Appveyor builds, which have been failing for your master as well. I would try to fix those, but I don't have a Windows development environment available.